docs/src/layouts/articleLayout.tsx

```diff
@@ -8,7 +8,15 @@ import SEO from '../components/seo'
 import { graphql } from 'gatsby'
 import MDXRenderer from 'gatsby-plugin-mdx/mdx-renderer'
 
-type ArticleLayoutProps = ArticleQueryData & RouterProps
+const DEFAULT_TOC_DEPTH = 3
```

Comment:
Default TOC depth is 3. That's voluntarily a pretty high number so that we have as many details as possible, until it becomes too much and then we can reduce it using the `tocDepth: number` mdx header property